driver-core/src/test/functional/com/mongodb/operation/AggregateToCollectionOperationSpecification.groovy

```diff
@@ -98,6 +99,25 @@ class AggregateToCollectionOperationSpecification extends OperationFunctionalSpe
         operation.getCollation() == defaultCollation
     }
 
+    def 'should set optional values correctly (with read concern)'(){
```

Comment:
Please add a test scenario to `should create the expected command` to test read Concern is included